datacube_stats/mangrove.py

```diff
@@ -0,0 +1,69 @@
+import logging
+import xarray as xr
+import numpy as np
+from osgeo import ogr
+from osgeo import gdal
+from osgeo import osr
+from datacube.model import Measurement
+from .statistics import Statistic
+
+_LOG = logging.getLogger(__name__)
```

Comment:
This isn't actually being used in this file, I think.

Reply:
OK... turned out I deleted all the log message...Don't know if I would need some in the future but better delete